specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json

```diff
@@ -1091,6 +1095,20 @@
         }
       }
     },
+    "ResourceMetadata": {
+      "description": "Recommendation resource metadata",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "description": "Azure resource Id of the assessed resource",
+          "type": "string"
+        },
+        "source": {
```

Comment:
Based on your samples, I think 'source' should be named something like 'sourceAssessmentId'. 'source' by itself doesn't tell the user much about its purpose.

Reply:
Source used in example is of assessment id but in future we  are planning to use other resource types as well (like app service recommendation resource type, sql recommendation resource type).
I understand your concern. Please suggest other name which is generic enough to cover scenario I mentioned above.